code_search/docker/ks/submit_code_embeddings_job.sh

```diff
@@ -41,9 +41,12 @@ names=(cluster dataDir functionEmbeddingsBQTable functionEmbeddingsDir modelDir
 source "${DIR}/parse_arguments.sh"
 source "${DIR}/initialize_kubectl.sh"
 
+# BQ doesn't support dash. Replace with underscore
+functionEmbeddingsBQTable=$(echo "${functionEmbeddingsBQTable}" | tr - _)
+
 # Apply parameters
 ks param set ${component} dataDir ${dataDir} --env ${ksEnvName}
-ks param set ${component} functionEmbeddingsBQTable ${functionEmbeddingsBQTable} --env ${ksEnvName}
+ks param set ${component} functionEmbeddingsBQTable ${project}:${functionEmbeddingsBQTable} --env ${ksEnvName}
```

Comment:
Shouldn't project be part of the functionEmbeddingBQTable name? i.e. shouldn't functionEmbeddingsBQTable be the full table spec?

Reply:
yeah i agree it should be. however the line above

> functionEmbeddingsBQTable=$(echo "${functionEmbeddingsBQTable}" | tr - _)

will replace the project from code-search-demo to code_search_demo. so i separate them to workaround the issue. 

I revert the change since I find some other parameter placeholder i can use for specifying the BQ table name.